renovate.json

```diff
@@ -83,6 +83,7 @@
         "pipenv",
         "poetry",
         "pnpm",
+        "protobuf",
```

Comment:
wrong

Reply:
the renovate depname is actually protobuf

Reply:
added both just in case

Reply:
no, `depName=protoc`. it's coming from the regex manager. `install-tool <depName> <currentValue>`